src/compiler/scala/tools/nsc/ast/parser/MarkupParsers.scala

```diff
@@ -172,20 +173,19 @@ trait MarkupParsers {
     }
 
     def appendText(pos: Position, ts: Buffer[Tree], txt: String): Unit = {
```

Comment:
It's not significant because appendText isn't invoked with empty text: either tag < follows and no text node, or literal { follows and text is at least a brace.